internals/cli/cli.go

```diff
@@ -353,7 +348,7 @@ func errorToMessage(e error) (normalMessage string, err error) {
 	return msg, nil
 }
 
-func getEnvPaths() (pebbleDir string, socketPath string) {
+func GetEnvPaths() (pebbleDir string, socketPath string) {
```

Comment:
If we do add `client.NewConfigFromEnv`, this probably doesn't need to be exported anymore.

Reply:
We would still need this to be exported so we can `GetEnvPaths()` from the `client` package though, am I wrong?

Reply:
Ah, true, I wasn't paying close enough attention that `GetEnvPaths` is in `cli` but `NewConfigFromEnv` is in `client`.

Actually, maybe `GetEnvPaths` should move to the `client` package? It's (sort of) a client concern, or half of it is. :-) Or maybe we should have `client.SocketPath()` and then just have an internal `cli.pebbleDir()` for the `run` command. Though I realize that will mean a tiny bit of duplication of the `PEBBLE` logic -- but it may be conceptually cleanest.

Oh, either way we should probably drop the `Get` prefix.

Reply:
This is clearly an internal helper function with API not intended to be exposed. Let's please not publish it.

Also, as a more general note: I realize that most of the work on going in your team is about exposing functionality, but it feels like too often the reviews are heavy on "Please don't expose things that were built privately without considering how to make them public.". Maybe some of these could be anticipated by internal reviews inside the team itself.